src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
 {
     /// <summary>
     /// The MembershipModeration class is intended to identify
```

Comment:
GitHub won't let me comment on the exact line but we shouldn't using the "GroupId" class in extension data -- we tell people not to do that in the documentation. Let's use a string instead.